packages/electron-builder-squirrel-windows/template.nuspectemplate

```diff
@@ -19,6 +19,7 @@
     <file src="*.pak" target="lib\net45" />
     <file src="*.exe.config" target="lib\net45" />
     <file src="*.exe.sig" target="lib\net45" />
+    <file src="*_ExecutionStub.exe" target="lib\net45" />

```

Comment:
Why is this now being added?

Reply:
Since electron-builder now generates and signs *_ExecutionStub.exe, adding this configuration will move the generated *_ExecutionStub.exe to the lib\net45 directory.